internal/request_builder.go

```diff
@@ -20,21 +21,27 @@ func NewRequestBuilder() *HTTPRequestBuilder {
 	}
 }
 
-func (b *HTTPRequestBuilder) Build(ctx context.Context, method, url string, request any) (*http.Request, error) {
-	if request == nil {
-		return http.NewRequestWithContext(ctx, method, url, nil)
+func (b *HTTPRequestBuilder) Build(ctx context.Context,
+	method, url string, body any, header http.Header) (req *http.Request, err error) {
```

Comment:
nit: maybe let's go one-argument-per-line style here, it's a bit longer but seems cleaner. What do you think?

```
func (b *HTTPRequestBuilder) Build(
	ctx context.Context,
	method,
	url string,
	body any,
	header http.Header
)
```

Reply:
@sashabaranov 
> one-argument-per-line style

Looks Good! I will update to this style.

Reply:
DONE!